### DISCUSSION TITLE
AG backup

### DISCUSSION THREAD

**Discussion by nmo0ory** [olahallengren/sql-server-maintenance-solution#601] [Questions & Answers]
Hello,
i would like to ask about the AG backup 
how can i setup the script to do so ? 
i saw some people talking about it but its old topic and now it seem there is option for AG name 
what it would do and how can i do it exactly?

**Comment by olahallengren** [comment]
There are some questions and answers about this in the Frequently Asked Questions (search for "an Availability Group").
https://ola.hallengren.com/frequently-asked-questions.html
